### ISSUE TITLE
Blocking a background image (+link)

### ISSUE THREAD

**Issue by ajolivet** [gorhill/uBlock#715]
Hi,

I'm trying to block the background image on this site: http://www.hltv.org/

Apparently there's also a link on this image.

I have the url of the image: http://static.hltv.org//images/alphabg.jpg

However, whenever I try to block it via contextual menu, it's impossible to select it.

Is it a normal behavior? If not, is it planned to be fixed?

Thanks for your time reading this, and thanks a lot for the work you put in this software.

**Comment by weakptr** [comment]
My Rules: 
`no-popups: www.hltv.org true`

My filters:
`||static.hltv.org//images/alphabg.jpg$image,first-party`

**Comment by ajolivet** [comment]
Thanks a lot, it worked.

**Comment by Maruchin** [comment]
Hello again. The site (www.hltv.org) has started to use an inline script (highlighted here) http://i.imgur.com/KaBnsdA.png so whenever you accidentally click on the background image, you are lead to an advertisement. 

I'm struggling quite a lot with understanding the wiki. I've tried my luck with dynamic filtering and I've added a rule that should disable inline-scripts, but for some reason it doesn't work. 

Under "my filters" I've added ||hltv.org^$inline-script yet this script still loads. Am I doing something incorrect, or are they countering ublock?
